ce/cli/commands/activate.ts

```diff
@@ -39,6 +41,11 @@ export class ActivateCommand extends Command {
     debug(i`Deactivating project ${projectFile(projectManifest.metadata.context.file)}`);
     await session.deactivate();
 
-    return await activateProject(projectManifest, this.commandLine);
+    return await activateProject(projectManifest, {
```

Comment:
Wait, didn't my change here destroy the ability to figure out what filters are used for activation? This is the part that I thought was broken before and expected CR feedback against in order to fix.

Reply:
I couldn't find anything at all wrong with it. conditional demands works fine.

Reply:
How do those values get passed in then? I assumed they came from passing the commandline object here and I'm not doing that anymore

Reply:
The `session` object contains all the command line values (`session.context`).